index.html

```diff
@@ -227,7 +227,7 @@ <h2>Scope</h2>
             </p>
 
             <p>
-                There are a variety of use cases in the area of Linked Data, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, or consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. (See the <a href="explainer.html#usage">use cases</a> for more examples.) In order to achieve these use cases, a standard way to process <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> is needed. 
+                There are a variety of use cases in the area of Linked Data, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, or consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. (See the <a href="explainer.html#usage">use cases</a> for more examples.) In order to achieve these use cases, a standard way is needed to process the underlying graph serialized in <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> that is independent of the serialization itself.
```

Comment:
Not only does this wording still implies that RDF Datasets contain a single graph, but it also gives the impression that "RDF Dataset" denotes the concrete syntax (i.e. the serialisation) while it actually denotes the abstract syntax (i.e. the graphs).

```suggestion
                There are a variety of use cases in the area of Linked Data, such as <a href="https://www.w3.org/TR/vc-data-model">Verifiable Credentials</a>, the publication of biological and pharmaceutical data, or consumption of mission critical RDF vocabularies that depend on the ability to verify the authenticity and integrity of the data being consumed. (See the <a href="explainer.html#usage">use cases</a> for more examples.) In order to achieve these use cases, a standard way is needed to process the underlying graphs contained in <a href="https://www.w3.org/TR/rdf11-concepts/#section-dataset">RDF Datasets</a> that is independent of the serialization itself.
```